tests/ci/Jenkinsfile

```diff
@@ -13,7 +13,7 @@ pipeline {
                 axes {
                     axis {
                         name 'TEST_NAME'
-                        values 'control', 'regional_control', 'cpld_control_nowave_noaero_p8'
+                        values 'control_p8', 'cpld_control_nowave_noaero_p8'
```

Comment:
@jkbk2004 May I ask why we remove the "regional_control" test?

Reply:
sure! @zach1221 and I will follow up with creating issue.